src/cli.ts

```diff
@@ -84,6 +84,15 @@ const argv = yargs
 		choices: [InitializeMode.Game, InitializeMode.Model, InitializeMode.Plugin, InitializeMode.Package],
 		conflicts: ["w", "no-analytics"],
 		type: "string",
+  })
+  
```

Comment:
Outstanding indentation issue here 😱 

```suggestion
	})

```